### PR TITLE
Fixed issue #17862: Invalid url in admin (usage of HTTP_HOST)

### PR DESCRIPTION
**Pull request by gabrieljenik**
Compiling of JS is pending

### CODE REVIEW

**Comment by Shnoulle** [comment]
>  Strangely : «Add question» is OK but not «add group» 

?

**Comment by gabrieljenik** [comment]
> > Strangely : «Add question» is OK but not «add group»
> 
> ?

Sorry, don't understand.
What do you mean?

**Comment by Shnoulle** [comment]
report : Go to manage admin survey :

1. Can create survey, can edit global settings a lot os OK
2. Add button question in structure goes to limesurvey.intra
3. Global survey settings tab goes all to limesurvey.intra

In structure : Add question is broken , not add group (i made an error in my report)

**BUT** : i can not reproduce today … on my local nginx …

**Review by c-schmitz — APPROVED** [review]

**Review by c-schmitz — CHANGES_REQUESTED** [review]
Code is okay, but CI is not green

**Comment by olleharstedt** [comment]
So it's finally happened. We need to apply a bug fix in adminsidepanel and the build system is FUBAR...

**Comment by olleharstedt** [comment]
I think Jessica tried to update to Vue 3, but you probably have to downgrade it to Vue 2 again to easily apply a new fix.

**Comment by olleharstedt** [comment]
Jessica is back next week, maybe we'll assign it to her then. Or it will have to be a team effort.

Also, death to dependabot.

**Comment by olleharstedt** [comment]
I moved the task to our internal Zoho project, we'll try to solve it.

**Comment by naiteon** [comment]
@gabrieljenik side menu has been fixed. If you find any problems reach out to me please.

**Comment by gabrieljenik** [comment]
@naiteon Thanks! What was it?
What are the next steps then? Should we merge master in here and recompile?

**Comment by olleharstedt** [comment]
Sorry, still broken. False alarm. :(

**Comment by gabrieljenik** [comment]
@naiteon Thanks! What was it?
What are the next steps then? Should we merge master in here and recompile?

**Comment by naiteon** [comment]
@gabrieljenik Hey Gabriel! I think you should stash your changes, rebase your branch against master, then apply the stashed changes and proceed with the PR.

**Comment by naiteon** [comment]
@gabrieljenik oh no, there is no need for recompiling it because the distribution files are included in the repo.

**Comment by olleharstedt** [comment]
> @gabrieljenik As for compiling the adminsidepanel, there is no need for recompiling it because the distribution files are included in the repo.

This fix changes the adminsidepanel src files, that's why they need to be recompiled.

**Comment by gabrieljenik** [comment]
Ready for re review!

**Review by Shnoulle — APPROVED** [review]

File: assets/packages/adminsidepanel/src/components/subcomponents/_questionsgroups.vue [hunk 1]
```diff
@@ -64,10 +64,11 @@ export default {
             this.active = [];
         },
         createFullQuestionLink() { 
-            if(LS.reparsedParameters().combined.gid) {
-                return LS.createUrl(this.createQuestionLink, {gid: LS.reparsedParameters().combined.gid}); 
+            if (LS.reparsedParameters().combined.gid) {
+                return this.createQuestionLink + '&gid=' + LS.reparsedParameters().combined.gid;
```
[the remark this answers is not in the document]
Comment (Shnoulle) [reply assets/packages/adminsidepanel/src/components/subcomponents/_questionsgroups.vue:68]: Really specific i think …

**Comment by gabrieljenik** [comment]
I think this can be merged. Seems tested and also reviewed? Please someone elese confirm it. Thanks

**Pull request merged**